codegen/projections/rails_json/lib/rails_json/parsers.rb

```diff
@@ -402,14 +398,11 @@ def self.parse(http_resp)
         end
         unless http_resp.headers['X-TimestampList'].nil? || http_resp.headers['X-TimestampList'].empty?
           data.header_timestamp_list = http_resp.headers['X-TimestampList']
-            .split(', ')
-            .map { |s| Time.parse(s) }
+            .split(',').each_slice(2).map { |v| Time.parse(v[0] + v[1])}
```

Comment:
Header list parser should maybe handle Time case too? It should probably also call methods in TimeHelper

Reply:
Yeah, move this case to a method in the header list parser since its pretty weird logic.

And used the TimeformatHelper for the other cases.